openfl-gramine/openfl.manifest.template

```diff
@@ -38,6 +38,10 @@ fs.mount.workspace.type = "chroot"
 fs.mount.workspace.path = "/workspace"
 fs.mount.workspace.uri = "file:/workspace"
 
+fs.mount.tmp.type = "chroot"
```

Comment:
does gramine support a tmpfs-style mount? if /tmp is used only for temporary files, it may be better to tell gramine that /tmp is in memory, so there will be no leftovers after its execution

Reply:
i looked in gramine docs and found the `tmpfs` filesystem type. i believe it can be more appropriate in this context